Styling bash code blocks in bash modules (#930)
arcus/education_modules: bash_command_line_101/bash_command_line_101.md

> module_id: bash_command_line_101

Reviewer: This comment refers to a line that isn't in the PR-affected code, so I can't comment there (see https://github.com/isaacs/github/issues/1655).  Ugh.  Anyway, in line 318, consider being a bit more explicit and saying "everything is text.  For example, this is the text we'd write to create a bolded hyperlink:

Author: added!

If you are new to bash scripting, it can take some adjustment to navigate by typing text. Once you get the hang of a few basic commands, this can be an easier way to deal with files and folders than dropdown menus and dragging icons from place to place. Even the language we use is a little different. In bash scripting you will hear folders referred to as **directories**.

> When you first open a shell, you will see a [**Command Prompt**](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt). This may look different for everyone, for example the two authors of this module see different things. One of us sees:

Reviewer: @drelliche what you're showing here includes the MAC addresses of your computers (the alphanumeric name CHOP gives its macbooks at least is the MAC address).  Not sure of the security implications but might be good to suppress in future?

> When you first open a shell, you will see a [**Command Prompt**](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt). This may look different for everyone, for example the two authors of this module see different things. One of us sees:

Reviewer: ```suggestion
When you first open a shell, you will see a [**Command Prompt**](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt). This may look different for everyone, for example the two authors of this module see different things. One of us sees the name of their computer followed by a couple of symbols and then their username, all ending with a money symbol.
```

```
(base) C02GM4SBQ05N:~ drelliche$ 
```

> and the other sees: 

Reviewer: ```suggestion
and the other sees similar information, but in a different order, with different extra symbols to separate the fields.
```

![Home directory of the shell in the default homebrew theme.](media/shell_home.png)

To enter a command into the shell, type it in where the (possibly blinking) cursor appears and press `Enter` or `return` on your keyboard.

### Navigation Commands

Just like when you navigate any file system, it is important to know where you are, see what files and folders are in that location, and know how move to other locations.

**Where am I?**

> The command `pwd`, which stands for **p**resent **w**orking **d**irectory, will return the path through your directory system to your current location. Here is what I see when I type the command `pwd`, you will see something similar:

Reviewer: ```suggestion
The command `pwd`, which stands for **p**resent **w**orking **d**irectory, will return the path through your directory system to your current location. Here is what I see when I type the command `pwd`.  Depending on your computer setup, you will probably see something similar:
```

```bash
(base) C02GM4SBQ05N:~ drelliche$ pwd
```

```+output
/Users/drelliche
```
**What is here?**

> The bash shell doesn't show the icons of the files and directories in your present working directory. Instead you will need to ask for a list of the directory's contents with the command `ls`. Here is what I see when I type the command `ls`, you will see something similar:

Reviewer: ```suggestion
The bash shell isn't like a graphical file browser:  it doesn't show images or icons that represent the files and directories in your present working directory. Instead, you will need to ask for a **l**i**s**t of the directory's contents with the command `ls`. Here is what I see when I type the command `ls`, you will see something similar:
```

Public
```

> 

Reviewer: @pm0kjp make a suggestion here to move this north of the first use of "+ output" and maybe enclose it in a highlight box.

> You can click on the word "output" to collapse the output boxes. Going forward, we will omit the command prompt from the input and just show you the code to enter, like:

Reviewer: Please also move this up on the page, it currently comes **after** the first 2 uses of "output".  Maybe line 196-ish?

```suggestion
At this point, you'll start to see output boxes that include the word "output", which show one example of the kind of output you might see when the command above is entered.  You can click on the word "output" to collapse the output boxes if you like. Going forward, we will omit the command prompt from the input and just show you the code to enter, like:
```

Author: Moved to immediately after the first occurrence of an output box.
